pages/your_tasks/version_control.md

```diff
@@ -69,16 +69,19 @@ Implementing version control in a research context involves more than just choos
    * Periodically clean up old branches and review access permissions
 
 ## How to cite this page
-Giacomo Peru, TO-DO
+To be added.
+
+## Credit
+This page draws on widely accepted practices in version control and research software engineering. While not based on specific external sources, it represents common principles found in many educational resources in this field.
```

Comment:
such as...? I think this statement should be added when we have taken/reinterpreted text from another source. If it has not, then maybe not include it?